tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -849,10 +853,7 @@ public function userTheSpaceShouldContainEntries(
 		TableNode $expectedFiles
 	): void {
 		$space = $this->getSpaceByName($user, $spaceName);
-		$this->theUserListsTheContentOfAPersonalSpaceRootUsingTheWebDAvApi(
-			$user,
-			$spaceName
-		);
+		$this->featureContext->setResponse($this->propfindSpace($user, $spaceName));
```

Comment:
It is not recommended to use `setResponse` in Then steps (it will mess up if we have multiple Then steps for When action).
But the refactor looks big in this case. so let it be for now.

Reply:
Alright. Will consider about that from next time.